src/helics/application_api/FederateInfo.cpp

```diff
@@ -385,12 +386,14 @@ std::unique_ptr<helicsCLI11App> FederateInfo::makeCLIApp()
            },
            "Specify the port number to use")
         ->check(CLI::PositiveNumber);
-    app->add_option("--localport", localport, "Port number to use for connections to this federate")
+    app->add_option("--localport,--localPort,--local_port",
+                    localport,
+                    "Port number to use for connections to this federate")
         ->ignore_underscore();
     app->add_flag("--autobroker",
                   autobroker,
                   "tell the core to automatically generate a broker if needed");
-    app->add_option("--key,--broker_key",
+    app->add_option("--broker_key,--brokerkey,--brokerKey",
```

Comment:
@trevorhardy  I am going to take these extra ones out,  they just add overhead to the checking since we have the ->ignore_underscore and ->ignore_case already.

Reply:
That's fine.